### DISCUSSION TITLE
i need help

### DISCUSSION THREAD

**Discussion by Feva4989** [emailjs-com/help#142] [Q&A]
can some one please tell me why my email js is not working please email me at l.lewisgrdon@gmail.com if you are able to help me

**Comment by xr0master** [comment]
If you have any questions, you can ask them here.
We have to close this uninformative request.

**Comment by Feva4989** [comment]
hello are you able to help

**Comment by Feva4989** [comment]
here is all of my code for my project could you please look into it for me
